specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/afdx.json

```diff
@@ -3978,6 +3978,190 @@
+      "post": {
+        "tags": [
+          "Migration"
+        ],
+        "description": "Migrates classic profile to new frontdoor profile.",
+        "operationId": "Profile_Migrate",
+        "x-ms-examples": {
+          "Profile_Import": {
+            "$ref": "./examples/Profile_Migrate.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "MigrateInput",
+            "in": "body",
+            "description": "Input parameters for the import operation.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/MigrateInput"
+            }
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/MigrateResult"
+            }
+          },
+          "default": {
+            "description": "CDN error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/AfdErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cdn/profiles/{profileName}/migrationCommit": {
+      "post": {
+        "tags": [
+          "Migration"
+        ],
+        "description": "Commit the migrated Azure Frontdoor(Standard/Premium) profile.",
+        "operationId": "Profiles_MigrationCommit",
+        "x-ms-examples": {
+          "Profiles_MigrationCommit": {
+            "$ref": "./examples/Profiles_MigrationCommit.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "profileName",
+            "in": "path",
+            "description": "Name of the CDN profile which is unique within the resource group.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted and the operation will complete asynchronously."
+          },
+          "default": {
+            "description": "CDN error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/AfdErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cdn/profiles/{profileName}/migrationRollback": {
```

Comment:
will need confirm from @jessie and @gunjan if we still keep the migration rollback option.